webrtc/RTCPeerConnection-remote-track-mute.https.html

```diff
@@ -39,7 +39,7 @@
     });
   });
   await exchangeOfferAnswer(pc1, pc2);
-  await unmuteResolver.promise;
+  await unmuteResolver;
```

Comment:
Hum, this change and some others might actually be needed as part of PR 14417.
I'll try to fix that.

Reply:
IIRC we could also make it backward compatible.